Library/Homebrew/tap.rb

```diff
@@ -870,7 +870,7 @@ def self.ensure_installed!
 
   sig { returns(String) }
   def remote
-    super if installed? || Homebrew::EnvConfig.no_install_from_api?
+    super if Homebrew::EnvConfig.no_install_from_api?
```

Comment:
Why not
```suggestion
    super if installed? && Homebrew::EnvConfig.no_install_from_api?
```
?

Genuine question; not a rhetorical one suggesting using `&&` is more correct.

Reply:
I think it's clearer and more consistent to not make this behaviour change in either case based in whether it's installed or not.